test/e2e/ha/scale_core_contrail_services_test.go

```diff
@@ -0,0 +1,336 @@
+package ha
```

Comment:
I'm not sure this file should go into master

Reply:
I believe that's why HA CI job failed

Reply:
Ok, some error during rebase